tests/suite/test_suite.py

```diff
@@ -32,10 +34,36 @@
     recovery.test,
 ]
 
-
-full_suite = []
-full_suite.extend(suite_rekey_recovery)
-full_suite.extend(suite_membership_recovery)
+all_tests_suite = [
```

Comment:
What about we use `dir()` on all the local imports and grep for all the methods that start with `test_`? We could even add a `@skip` decorator if we don't want a certain test not to be included.

Reply:
We could do this - we could even make our `@reqs.description` decorator an opt-in, rather than adding an `@skip` opt-out.

However I'm not sure its worth it. I think its valuable to have an explicit list of all the tests we think we're running. Sure we can remove some user error in failing to register new tests here, but on the other hand modifying the list (eg - for debugging I want a suite of `[recovery, recovery, recovery, view_history]`) becomes more difficult as the available/present test names aren't visible.

Reply:
Running the test suite with a new `--list-tests` could list all the available tests and we could even pass the name of test cases from the command line, e.g. `--test-cases recovery,recovery,recovery,view_history` to run specific tests. Running the test suite with no `--test-cases` or `--test-suites` filters would mean running all the `test_` included from `test_suite`. 

We could still define custom sub-suites in code though.

Reply:
^ Not in the scope of this PR. Just discussing what I'd been thinking about the evolution of our test suite.

Reply:
Pretty soon that's going to be reimplementing all of the pytest runner :)

Reply:
Yeah you're right - with inspected suite names and test case names, we can build precise tests on the command-line. That's cool, but going to leave it for future PRs.